challenges/classes.js

```diff
@@ -1,7 +1,40 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+  constructor(attributes) {
+    this.length = attributes.length;
+    this.width = attributes.width;
+    this.height = attributes.height;
+  }
+  volume() {
+    return this.length * this.width * this.height;
+  }
+  surfaceArea() {
+    return (
+      2 *
+      (this.length * this.width +
+        this.length * this.height +
```

Comment:
what you have done is fine but yo can/should also use brackets to mark the separation  between 2 operator. so, it's easier to read. you can do something like

`return 2 * ((this.length * this.width) + (this.length * this.height) + (this.width * this.height));`